Removed redundancy

Linda-Editor opened the pull request:
To make the page more consistent, I removed the labels in the limit column because they are defined in the description.

opbld32 commented:
Docs Build status updates of commit _[23c8fc6](https://github.com/microsoftgraph/microsoft-graph-docs/commits/23c8fc6b7a68340689acd3e6ca9f4d4db9082cc1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/connecting-external-content-api-limits.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/connecting-external-content-api-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/connecting-external-content-api-limits?branch=pr-en-us-16802) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205061637528688-16802%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-06T16%3A40%3A19Z&sr=b&sp=r&sig=wpNDmlqSTpHttOTXj8iy9V%2F71orjzwJmvpIIaAXEDN0%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Lauragra commented on concepts/connecting-external-content-api-limits.md:
```diff
@@ -15,22 +15,22 @@ This article describes implementation and operational limits for Microsoft Graph
 | --------- | --------------- |
-| **10 connections** | The maximum number of [connection](/graph/api/resources/externalconnectors-externalconnection?view=graph-rest-1.0&preserve-view=true) resources per Microsoft 365 tenant. |
-| **700,000 items** | The maximum number of [items](/graph/api/resources/externalconnectors-externalitem?view=graph-rest-1.0&preserve-view=true) per connection. |
+| **10** | The maximum number of [connection](/graph/api/resources/externalconnectors-externalconnection?view=graph-rest-1.0&preserve-view=true) resources per Microsoft 365 tenant. |
```
@Linda-Editor , I think we can look for a different way to make these tables more clear. I don't think removing the labels makes these tables scannable/easy to read. What if we do something like this:

|Limit type | Maximum number |
|-----------|----------------------|
|Connection resources per Microsoft 365 tenant | 10 |
| Items per connection | 700,000 |
| Connection byte size | 70 GB |

Linda-Editor replied:
You're right. That's much better. I'll make these changes.

opbld31 commented:
Docs Build status updates of commit _[202871c](https://github.com/microsoftgraph/microsoft-graph-docs/commits/202871c608b699104b1d6d6ab74897fec3310902)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/connecting-external-content-api-limits.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/connecting-external-content-api-limits.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/connecting-external-content-api-limits?branch=pr-en-us-16802) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C9%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205091634564601-16802%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-09T16%3A36%3A50Z&sr=b&sp=r&sig=jaXypifx6NxgAWcyHKN34lkY2rCAELxZanQhfYhYoAQ%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Linda-Editor commented:
FYI: I changed "Maximum number" to "Maximum" because sometimes it was a number and sometimes it was a size, and then I changed it to "Limit" (which was the original heading) because "Maximum" didn't really work by itself. Let me know if you have any questions.
